### PR TITLE
Ft routine full screen

### PR DESCRIPTION
**Pull request by chidiogo-akunne**
#### What does this PR do?

Show full routine details
IMG_2740
IMG_2739

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

javascript
git checkout develop

git pull origin develop

yarn install

yarn test

Using an emulator: Launch the app, and you should see the home screen after loading.

### CODE REVIEW

**Comment by ogbodo** [comment]
Why two screen shots of same intention. Is it on purpose?

File: src/screens/regimen/regimen_routine/index.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { ScrollView, StatusBar, Platform, View } from 'react-native';
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/index.tsx:2]: remove unused imports => Platform, View

File: src/screens/regimen/regimen_routine/styles.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
+import styled from 'styled-components/native';
+import Constants from 'expo-constants';
+
+import applyScale from '../../../utils/applyScale';
+
+export const SplashTopStatusBar = styled.View`
+  width: 100%;
+  height: ${Constants.statusBarHeight}px;
+  position: absolute;
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/styles.ts:9]: Always try using flex to arrange component in other  take full advantage of responsiveness across all screen sizes.

File: src/screens/regimen/regimen_routine/styles.ts [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  justify-content: center;
+  align-items: center;
+  border-radius: 25px;
+  background-color: ${({ theme }) => theme.colors.BG_BLUE_GREEN_COLOR};
+`;
+
+export const BackgroundCover = styled.Image`
+  flex: 1;
+  width: 100%;
+  position: absolute;
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/styles.ts:35]: Always try using flex to arrange component in other take full advantage of responsiveness across all screen sizes.

**Review by ogbodo — CHANGES_REQUESTED** [review]

File: src/screens/regimen/regimen_routine/index.tsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+const HEADER_EXPANDED_HEIGHT = 300;
+
+const products = RecommendedProducts;
+
+interface RegimenRoutineScreenProp extends NavigationInterface {
+  testID?: string;
+}
+
+export default function RegimenRoutineScreen(props: RegimenRoutineScreenProp) {
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/index.tsx:44]:
```suggestion
export default function RegimenRoutineDetailScreen(props: RegimenRoutineScreenProp) {
```

File: src/screens/regimen/regimen_routine/styles.ts [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin: 4% 0 2%;
+`;
+
+export const Night = styled.Text`
+  color: ${({ theme }) => theme.colors.FONT_BLUE_COLOR};
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
+  font-size: ${({ theme }) => theme.fonts.SMALL_SIZE};
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/styles.ts:50]: Please add unit here

File: src/screens/regimen/regimen_routine/styles.ts [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+  margin-left: 1%;
+  text-transform: uppercase;
+`;
+
+export const TitleText = styled.Text`
+  font-size: ${({ theme }) => theme.fonts.LARGE_SIZE + 5}px;
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/styles.ts:57]: here too

File: src/screens/regimen/regimen_routine/styles.ts [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+  color: rgba(0, 0, 0, 0.9);
+  margin-bottom: 3%;
+`;
+
+export const Description = styled.Text`
+  text-align: center;
+  width: 65%;
+  font-family: ${({ theme }) => theme.fonts.CORMORANT_MEDIUM};
+  font-size: ${({ theme }) => theme.fonts.LARGE_SIZE};
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/styles.ts:66]: here too

File: src/screens/regimen/regimen_routine/styles.ts [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+  text-align: center;
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
+  font-size: ${({ theme }) => theme.fonts.LARGE_SIZE + 5}px;
+  color: ${({ theme }) => theme.colors.FONT_DARK_COLOR};
+  text-transform: uppercase;
+`;
+
+export const Price = styled.Text`
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
+  font-size: ${({ theme }) => theme.fonts.MEDIUM_SIZE};
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/styles.ts:93]: here too

File: src/screens/regimen/regimen_routine/styles.ts [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+  padding-top: 10px;
+`;
+
+export const ProductImage = styled.Image`
+  margin: 8% 0;
+`;
+
+export const Name = styled.Text`
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
+  font-size: ${({ theme }) => theme.fonts.SMALL_SIZE};
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/styles.ts:114]: here too, and every other places that you are supposed to add it.

File: src/screens/regimen/regimen_routine/styles.ts [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+  text-transform: uppercase;
+`;
+
+export const Price = styled.Text`
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
+  font-size: ${({ theme }) => theme.fonts.MEDIUM_SIZE};
+  color: rgba(0, 0, 0, 0.7);
+`;
+
+export const PriceSub = styled.Text`
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/styles.ts:97]:
```suggestion
export const PriceSuperscript = styled.Text`
```

File: src/libs/recommendedProducts.ts [hunk 11]
```diff
@@ -0,0 +1,26 @@
+export const RecommendedProducts = [
+  {
+    price: '$2',
+    subPrice: '74',
```
Comment (ogbodo) [inline src/libs/recommendedProducts.ts:4]:
```suggestion
    priceSuperscript: '74',
```

**Review by ogbodo — CHANGES_REQUESTED** [review]

File: src/screens/regimen/regimen_routine/styles.ts [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+`;
+
+export const TimeCover = styled.View`
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin: 4% 0 2%;
+`;
+
+export const Night = styled.Text`
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/styles.ts:47]:
```suggestion
export const Period = styled.Text`
```

File: src/screens/regimen/regimen_routine/styles.ts [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+  font-size: ${({ theme }) => theme.fonts.LARGE_SIZE + 5}px;
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
+  color: rgba(0, 0, 0, 0.9);
+  margin-bottom: 3%;
+`;
+
+export const Description = styled.Text`
+  text-align: center;
+  width: 65%;
+  font-family: ${({ theme }) => theme.fonts.CORMORANT_MEDIUM};
```
Comment (ogbodo) [inline src/screens/regimen/regimen_routine/styles.ts:65]:
```suggestion
  font-family: ${({ theme }) => theme.fonts.CORMORANT_REGULAR};
```

**Review by ogbodo — CHANGES_REQUESTED** [review]

File: src/screens/regimen/regimen_routine/index.tsx (hunk 1 above)
Comment (chidiogo-akunne) [reply src/screens/regimen/regimen_routine/index.tsx:2]: Done

**Review by ogbodo — APPROVED** [review]

**Pull request closed without merging**